packages/logger/tests/e2e/sampleRate.decorator.test.FunctionCode.ts

```diff
@@ -2,27 +2,39 @@ import { Logger } from '../../src';
 import { APIGatewayProxyEvent, Context } from 'aws-lambda';
 import { LambdaInterface } from '@aws-lambda-powertools/commons';
 
-const SAMPLE_RATE = parseFloat(process.env.SAMPLE_RATE);
-const LOG_MSG = process.env.LOG_MSG;
+const SAMPLE_RATE = parseFloat(process.env.SAMPLE_RATE || '0.1');
+const LOG_MSG = process.env.LOG_MSG || 'Hello World';
 
 const logger = new Logger({
   sampleRateValue: SAMPLE_RATE,
 });
 
 class Lambda implements LambdaInterface {
+  private readonly logMsg: string;
+
+  public constructor() {
+    this.logMsg = LOG_MSG;
+  }
+
   // Decorate your handler class method
   @logger.injectLambdaContext()
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
   public async handler(event: APIGatewayProxyEvent, context: Context): Promise<{requestId: string}> {
-    logger.debug(LOG_MSG);
-    logger.info(LOG_MSG);
-    logger.warn(LOG_MSG);
-    logger.error(LOG_MSG);
-
+    this.dummyMethod();
+    
     return {
       requestId: context.awsRequestId,
     };
   }
+
+  private dummyMethod() : void {
```

Comment:
(optional) Calling this `dummyMethod` is a bit misleading.  What about `printLogInAllLevels`?

The reason is that these logs are used for assertion later so they are important.